src/pybind/mgr/dashboard/model/feedback.py

```diff
@@ -0,0 +1,56 @@
+# # -*- coding: utf-8 -*-
+from enum import Enum
+
+class Feedback:
+    project_id: int
+    tracker_id: int
+    subject: str
+    description: str
+    severity: str
+    status: int
+
+    class Project(Enum):
+        dashboard = 46
+        block = 9 #rbd
+        object = 10 # rgw
+        file_system = 13 # cephfs
+        ceph_manager = 46
+        orchestrator = 42
+        ceph_volume = 39
+        core_ceph = 36 # rados
+
+    class Tracker(Enum):
+        bug = 1
+        feature = 2
+
+    class Severity(Enum):
+        minor = '3 - Minor'
+        major = '2 - Major'
+        critical = '1 - Critical'
+
+    class Status(Enum):
+        new = 1
+
+    def __init__(self, project_id, tracker_id, subject, description, severity):
+        self.project_id = int(project_id)
+        self.tracker_id = int(tracker_id)
+        self.subject = subject
+        self.description = description
+        self.severity = severity
+        self.status = self.Status.new.value
```

Comment:
Since Status is a nested class and also static to the `Feedback` class I would change it to this:
```suggestion
        self.status = Feedback.Status.new.value
```